public/views/content/d3.html

```diff
@@ -0,0 +1 @@
+<nvd3 options="options" data="data"></nvd3>
```

Comment:
A little bit of both - nvd3 is basically a library in between this one and d3, so I think that might be where it originally came from